test/partiql-tests-runner/src/test/kotlin/org/partiql/runner/ConformanceTestBase.kt

```diff
@@ -10,6 +11,7 @@ abstract class ConformanceTestBase<T, V> {
     abstract val runner: TestRunner<T, V>
 
     // Tests the eval tests with the Kotlin implementation
+    @Timeout(value = 100, threadMode = Timeout.ThreadMode.SEPARATE_THREAD)
```

Comment:
Unit is second. This is not a performance test. This is for stop long running tests during development process in eval engine. This number can be smaller, but to account for the cold start time and frustration of GitHub runner, decided to make this number a bit larger than needed.

Reply:
Could you please add this comment in the code itself?